src/Weather.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+export default class Weather extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      weather: {}
+    }
+  }
+
+  getWeatherConditions = async () => {
+    const link = `${process.env.REACT_APP_SERVER_URL}/weather`;
```

Comment:
When I take a look at deployed app, I am getting 404 and the request URL is as follows (shown in Network tab using Chrome Dev Tools, under `Headers` tab): 

`https://rs-city-explorer.netlify.app/undefined/weather`

Grab a TA to help ensure we have the proper server URL used and you will need use query here as well in this url. Something like:
const url = `${process.env.REACT_APP_SERVER_URL}/weather?searchQuery=${whatever is in state for display name}&lat=${lat}&lon=${lon}`;